platform-ajax/src/test/java/com/softicar/platform/ajax/AjaxTestResources.java

```diff
@@ -16,6 +16,6 @@ public interface AjaxTestResources {
 
 	IResourceSupplierFactory FACTORY = new DomResourceSupplierProxyFactory(AjaxTestResources.class);
 
-	IResourceSupplier TEST_PNG = FACTORY.create("test.png");
-	IResourceSupplier TEST_TIFF = FACTORY.create("test.tiff");
+	IResourceSupplier PNG_TEST = FACTORY.create("png-test.png");
```

Comment:
This renaming was necessary because otherwise the resources enumerated here had colliding keys, which in turn resulted in a pseudo-(non-)determinism in `AjaxDomEngineTest`.

Reply:
nice